README.md

```diff
@@ -82,6 +82,10 @@ You can pass updates to the configuration through the command line like so:
 
 To extract alignments from the model run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt`.
 
+To output alignments in text\_grid, run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt --tg utt_count`, where `utt_count` is the number of utterances in your dataset.
```

Comment:
```suggestion
To visualize some of your alignments and inspect them with [Praat](https://www.fon.hum.uva.nl/praat/), run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt --tg utt_count`, where `utt_count` is the number of sample alignments you want to generate TextGrids for.
To output alignments in text\_grid format for your entire dataset, run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt --tg utt_count`, where `utt_count` is the number of utterances in your dataset.
```

Reply:
This revision might make it more clear why you would want to generate TextGrids, and also gives some motivation for why you might want to have `utt_count` be lower than the total number of utterances in the dataset.

Reply:
Yes you're definitely correct!  The revision is much more clearer! Thanks Aidan!